lib/tools/e2b/sandbox.ts

```
@@ -1,8 +1,9 @@
-import { createSupabaseAdminClient } from '@/lib/server/server-utils';
 import { Sandbox } from '@e2b/code-interpreter';
 import { safeWaitUntil } from '@/lib/utils/safe-wait-until';
+import { ConvexHttpClient } from 'convex/browser';
+import { api } from '@/convex/_generated/api';
 
-const supabaseAdmin = createSupabaseAdminClient();
+const convex = new ConvexHttpClient(process.env.NEXT_PUBLIC_CONVEX_URL!);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)